src/components/ErrorBanner.jsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react'
+import React from "react";
 
-const ErrorBanner = () => {
+function ErrorBanner() {
   return (
-    <div>ErrorBanner</div>
-  )
+    <div
+      className=" flex justify-center items-center mb-3 rounded-md w-auto
```

Comment:
Check and see if some of these classes are redundant and don't do anything, no need for them as they will just increase the size of the website